assistant_bot/commands/cmd_note/cmd_add_note.py

```diff
@@ -36,10 +36,8 @@ def execute(self, args, book):
                 record.add_note(note)
                 book[name] = record
 
-            msg = "Note does not exist."
+            msg = "Note added to the contact."
```

Comment:
Note added: It is necessary here.
Our contacts and notes are not connected to each other